src/components/Slider/styles.ts

```diff
@@ -0,0 +1,79 @@
+import { InputHTMLAttributes } from "react";
+import styled from "styled-components";
+
+// Using require instead of import to avoid trouble with TS bunding and SVG types
+/* eslint-disable @typescript-eslint/no-var-requires */
+const bunnyHead = require("./svg/bunnyhead-main.svg");
```

Comment:
The reason for doing it like this is because I need the link to the svg in the css, I'm not using the svg as a react component. If you got a preferred solution for assets I could change it, but I don't thing react-svg will work :/